### PR TITLE
[EDU-4202] add serverless arch

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: Implement Serverless Architecture with Azion
+description: Learn how to leverage Azion's Edge Platform to create serverless applications. Replace legacy CDNs with edge-native content for improved speed, reliability, and cost efficiency.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx:3]:
```suggestion
description: Leverage Azion Edge Platform to create serverless applications. Replace legacy CDNs with edge-native content for improved speed, reliability, and cost efficiency.
```

File: src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+meta_tags: edge, applications, web applications, api, content delivery, cdn, serverless, Azion
+namespace: docs_arch_serverless_apps
+permalink: /documentation/architectures/edge-application/serverless-applications/
+---
+
+Serverless computing allows you to build and run applications without worrying about the underlying infrastructure. With Azion's Edge Platform, you can focus more on creating quality products and less on managing servers.
+The benefits of serverless are numerous: it offers scalability, cost-efficiency, and speed. By leveraging Azion's Edge Functions, you can implement your business logic closer to the end user, resulting in ultra-fast, responsive applications. Dive into this guide to explore the architecture and implementation of serverless applications on Azion.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx:10]:
```suggestion
Serverless computing allows you to build and run applications without worrying about the underlying infrastructure. With Azion Edge Platform, you can focus more on creating quality products and less on managing servers.

The benefits of serverless are numerous: it offers scalability, cost-efficiency, and speed. By leveraging Azion Edge Functions, you can implement your business logic closer to the end user, resulting in ultra-fast, responsive applications. Dive into this guide to explore the architecture and implementation of serverless applications on Azion.
```

File: src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+5. Operations requiring database transactions can be accomplished with **Edge SQL**.
+
+After all this process, the content is delivered to the client.
+
+---
+
+## Components
+
+- **Edge Application**: set up an edge application to configure delivery and cache policies, create rules to automate cache policy assignments based on content type, and determine how content is cached.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx:37]:
```suggestion
- **Edge Application**: allows you to set up an edge application to configure delivery and cache policies, create rules to automate cache policy assignments based on content type, and determine how content is cached.
```

File: src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- **Origins**:  allows you to customize the source address of your Edge Application content, the Host header of your application, the path, and the HMAC authentication credentials for protected services.
+- **Domains**: register a custom domain with Azion to deliver the edge application.
+- **Edge Storage**: a scalable and secure storage service designed to integrate object storage with the Azion Edge Platform using the S3 standard for object operations.
+
+---
+
+## Implementation
+
+1. Create an edge application:
+    - On Console, create an application using the + Create button on the homepage.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx:53]:
```suggestion
    - On Console, create an application using the **+ Create** button on the homepage.
```

File: src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    - Use the Azion API to create an edge application.
+    - Use the Azion CLI to create and deploy an edge application.
+2. Create a domain and associate it with the edge application: 
+- On Console, create a domain using the + Create button on the homepage.
+- Use the Azion API to create a domain.
+
+If you created an application through a template, the domain is created for you.
+> The edge application is accessible through the domain provided by Azion, which looks similar to: `ftd1cvntfl.map.azionedge.net` 
+Test and monitor: after configuring your content delivery, thoroughly test the content delivery to ensure it's functioning as expected and monitor incoming access.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx:62]:
```suggestion
3. Test and monitor: after configuring your content delivery, thoroughly test the content delivery to ensure it's functioning as expected and monitor incoming access.
```

File: src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+Make adjustments to caching policies, rules, or other configurations as needed based on performance metrics and user feedback.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx:63]:
```suggestion
- Make adjustments to caching policies, rules, or other configurations as needed based on performance metrics and user feedback.
```

File: src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+- **Tiered Cache**: module that creates an additional cache layer between the edge and your origin servers, reducing latency and infrastructure costs. 
+- **Load Balancer**: module designed to balance traffic across your origins, data centers, or cloud providers, mitigating network congestion and server overload.
+- **Edge Storage**: a scalable and secure storage service designed to integrate object storage with the Azion Edge Platform using the S3 standard for object operations.
+- **Origins**:  allows you to customize the source address of your Edge Application content, the Host header of your application, the path, and the HMAC authentication credentials for protected services.
+- **Domains**: register a custom domain with Azion to deliver the edge application.
+- **Edge Storage**: a scalable and secure storage service designed to integrate object storage with the Azion Edge Platform using the S3 standard for object operations.
+
+---
+
+## Implementation
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx:50]: remember to include links related to the steps

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**